core-java-modules/core-java-numbers-6/src/test/java/com/baeldung/javadoublevsbigdecimal/BigDecimalUnitTest.java

```diff
@@ -0,0 +1,49 @@
+package com.baeldung.javadoublevsbigdecimal;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class BigDecimalUnitTest {
+    private BigDecimal bigDecimal1;
+    private BigDecimal bigDecimal2;
```

Comment:
Having a look at the imports and spaces between statements seems the Baeldung formatter is not being used.

Reply:
Fixed